docs/current/cli/979595-reference.md

```diff
@@ -27,10 +27,35 @@ The options below can be used with all CLI commands.
 | `-s`, `--silent`      | Disable terminal UI and progress output                          |
 | `-h`, `--help`        | Show help text                                                   |
 | `--workdir`           | Define the host working directory (default `.`)                  |
-| --------------------- | ---------------------------------------------------------------- |
 
 ## Commands
 
+## dagger call
+
+:::note
+This command is currently under development and is therefore hidden in the CLI.
+:::
+
+Call a module function and print the result. When called:
+
+- on a container, the standard output is returned;
+- on a directory, the list of entries is returned;
+- on a file, the file contents are returned.
+
+### Usage
+
+```shell
+dagger call [function]
+```
+
+### Examples
+
+Call a function returning a container. The standard output of the container is returned.
+
+```shell
+dagger call build
```

Comment:
A build task usually produces an artifact that you want access to, so it's more adequate for `dagger dl`. Testing is more suitable as a common task here.
```suggestion
dagger call test
```